Libraries/Text/RCTText.m

```diff
@@ -237,7 +237,11 @@ - (BOOL)canBecomeFirstResponder
 - (BOOL)canPerformAction:(SEL)action withSender:(id)sender
 {
   if (action == @selector(copy:)) {
```

Comment:
This way the next responder could decide to have the copy item or not? Makes sense.